### ISSUE TITLE
TODO: Xiaomi native tracker list

### ISSUE THREAD

**Issue by hagezi** [hagezi/dns-blocklists#2437]
**Xiaomi native tracker list**

- [x] Determine tracking domains
- [x] Compile list
- [x] Integrate into the build system
- [x] Provide as test version
- [x] Integrate beta into Ultimate
- [x] Integrate beta into Pro++
- [x] Integrate beta into Pro

**Comment by hagezi** [comment]
BETA/Test version is available.

https://github.com/hagezi/dns-blocklists?tab=readme-ov-file#native

**Comment by hagezi** [comment]
Beta version was integrated into the Ultimate.

**Comment by brokoler** [comment]
Hello @hagezi
I'm using a Xiaomi product and the Ultimate list.

Today I discovered my toilet seat went offline :)
It's using following domain to connect to the cloud service: **ot.io.mi.com**
I'm not sure if this domain is also a tracking URL, are you able to further validate?

Kind regards

**Comment by hagezi** [comment]
@brokoler Thanks for testing and your support. An offline toilet seat is of course not an option. The domain will be removed immediately.

I just wanted to test who uses a smart toilet seat. Although I seriously wonder what a smart toilet seat does ... ;)

**Comment by hagezi** [comment]
The next stage, the native trackers, will be added to Pro++ in the next release.

**Comment by quiniapiezoelectricity** [comment]
FYI
https://github.com/TG-Twilight/AWAvenue-Ads-Rule/issues/38
TLDR Translation:
Basically blocking [a0.app.xiaomi.com] would prevent Xiaomi devices from backing up to Xiaomi Cloud, but this domain is also used for uploading the app list on your phone to Xiaomi (with instances of users based in China getting phone calls for installing "suspicious" apps).

Might be worth considering when adding the Xiaomi list to less aggressive lists

**Comment by hagezi** [comment]
Many thanks for the hint @quiniapiezoelectricity

**Comment by hagezi** [comment]
The next stage, the native trackers, will be added to Pro in the next release.

**Comment by ttinkywinky45** [comment]
**Hi 👽 ! If you are interested in ⚙️Advanced Phishing⚙️ and the ✨Evilginx3✨ tool then I offer you Google Gmail and Custom Phishlets (Paypal, Cryptos...) with a ✅Free Tutorial✅ which arrives if the 📈Telegram reaches 50 Subscribers📈
⏳ GOOGLE GMAIL PHISHLET 2024 WITH PROOF ⏳
 ->
⛔https://www.youtube.com/watch?v=nA47UknIE6A⛔
---
🔓 Ask your questions and a free tutorial from 50 subs on telegram !!!
->
📩https://t.me/Noremorse1011📩**

![evilginx_background](https://github.com/kgretzky/evilginx2/assets/144027798/0e448f68-ecdc-49a4-a4f4-23dbed7cb971)
